Src/PerformanceCollector/Shared/Implementation/WebAppPerformanceCollector/NormalizedCPUPercenageGauge.cs

```diff
@@ -0,0 +1,78 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation.WebAppPerformanceCollector
+{
+    using System;
+    using System.Globalization;
+
+    /// <summary>
+    /// Gauge that computes normalized CPU percentage utilized by a process by utilizing the last computed time.
```

Comment:
Mention in comment that Normalized means taking into account the # of CPU Cores for clarity.